package.json

```diff
@@ -7,6 +7,7 @@
     "all": "grunt all",
     "standalone": "grunt standalone",
     "wpt": "grunt wpt",
+    "node": "grunt node",
```

Comment:
This is confusing (already, before your change) because `npm run node` will compile `out-node` while `tools/run_node` will run tests under node.

The out-node build is quite niche and I think we should just leave it out of here. (If removed here, change to `npx grunt node` at [1].)

Reply:
I sort of agree, but also feel like not having something that matches the other two confusing. With the same argument, I would expect `node standalone` to run a server, not just build it.

Reply:
ack, no strong feeling.